src/blueapi/cli/cli.py

```diff
@@ -125,6 +128,22 @@ def get_devices(obj: dict) -> None:
     """Get a list of devices available for the worker to use"""
     client: BlueapiRestClient = obj["rest_client"]
     pprint(client.get_devices().dict())
+@controller.command(name="listen")
+@check_connection
+@click.pass_obj
+def listen_to_events(obj: dict) -> None:
+    """Listen to events output by blueapi"""
+    config: ApplicationConfig = obj["config"]
+    amq_client = AmqClient(StompMessagingTemplate.autoconfigured(config.stomp))
+
+    def on_event(context: MessageContext, event: Union[WorkerEvent, ProgressEvent, DataEvent],) -> None:
+        converted = json.dumps(event.dict(), indent=2)
+        print(converted)
+
+    print(f"Subscribing to all bluesky events from {config.stomp.host}:{config.stomp.port}")
+    with amq_client:
+        amq_client.subscribe_to_all_events(on_event)
+        input()
```

Comment:
Is it worth adding a message along the lines of 'Press enter to exit'?

Reply:
Wonderfully paradoxical sentence but yes